Changes

```diff
@@ -1,3 +1,14 @@
+0.53.6.2 (relative to 0.53.6.1)
+========
+
+Improvements
+------------
+
+- CustomAttributes : Added extraAttributes plug that can be set from CompoundData
+- StandardLightVisualiser : Setting locatorScale to 0 now avoids costly computations
+- MultiLineTextWidget : Fixed unicode handling in `selectedText()`
```

Comment:
These last 2 should be separated into a `Fixes` section.

Also, I forgot to mention this the last time (and the time before that), but we should include the original PR-to-master numbers associated with these (see the 0.53.5.0 release notes for example).

Reply:
Ah yeah, I had noticed that afterwards and I think I updated the Changes on master but left the maintenance branch as is. I've now added the PR numbers in 41be2a5.

Thanks!